tasks/configure_namespace.yml

```diff
@@ -74,6 +74,21 @@
   - source-containers
 
 
+- name: Create configmaps
+  k8s:
+    state: present
+    definition: "{{ lookup('template', 'templates/reactor_config_map.yml') | from_yaml }}"
+    validate:
+      fail_on_error: no
+      strict: yes
+  with_items:
+    - { name: "reactor-config-map", allowed_domains: "{{ allowed_domains }}"}
```

Comment:
This doesn't look very scalable, wouldn't be better to just pass 2 different config maps in playbook?

Reply:
configmaps moved to playbook